Vagrantfile

```diff
@@ -6,6 +6,196 @@ $NODEMEM=256
 # Overwrite host locale in ssh session
 ENV["LC_ALL"] = "en_US.UTF-8"
 
+BOOTSTRAP_SCRIPT=<<__EOF__
+#!/bin/bash
+# Bootstrap the Ansible Lightbulb workshop environment on a Vagrant box
+
+#
+# Bash shell settings: exit on failing commands, unbound variables
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#
+# Variables
+#
+# Color definitions
+readonly reset='\e[0m'
+readonly red='\e[0;31m'
+readonly yellow='\e[0;33m'
+readonly cyan='\e[0;36m'
+
+readonly workshop_user=vagrant
+
+readonly workshop_key="/home/${workshop_user}/.ssh/id_rsa"
+readonly workshop_inventory="/home/${workshop_user}/hosts.ini"
```

Comment:
I think this will conflict with #265 that @dagwieers submitted.

Reply:
Agreed, but merge this one first and I'll resolve the conflicts...